### PR TITLE
add OCP 4.13 cluster pool for stackrox

### PR DESCRIPTION
**Pull request by tommartensen**
Followed instructions at https://docs.ci.openshift.org/docs/how-tos/cluster-claim/

Secret: https://vault.ci.openshift.org/ui/vault/secrets/kv/show/selfservice/stackrox-stackrox-hive/stackrox-ocp-4-13-cluster-pool-credentials

created DNS zone + IAM user for that, see PR: https://github.com/stackrox/automation-iac/pull/5

cc @gavin-stackrox @davdhacs @BradLugo

### CODE REVIEW

**Comment by tommartensen** [comment]
@bear-redhat @droslean 

I see all tests passed - is there a way to test the new ClusterPool like https://docs.ci.openshift.org/docs/architecture/ci-operator/#testing-with-a-cluster-from-a-cluster-pool before merging?

File: clusters/hive/pools/stackrox/OWNERS [hunk 1]
```diff
@@ -0,0 +1,10 @@
+approvers:
```
Comment (gavin-stackrox) [inline clusters/hive/pools/stackrox/OWNERS:1]: Maybe add Misha and Marcin to get some EU TZ coverage?

Comment (tommartensen) [reply clusters/hive/pools/stackrox/OWNERS:1]: done

**Comment by tommartensen** [comment]
/cc @bbguimaraes 

Kindly review the PR. Is there a way to test the cluster pool before merging?

**Pull request merged**